docs/tutorial_tx_service_initiate_sign.md

```diff
@@ -194,6 +194,9 @@ print(signature.signature.hex())
 Signature is
 `0xc0df6a1b659d56d3d23f66cbd1c483467ea68a428fea7bbbe0a527d43d8681f616af33344035f36c08218718480374dada0fe6cdb266d0182a4225d0e9c227181b`.
 
+More than one signature (for different owners) can be sent, they should be appended and result should be an hex string `0x` prefixed. So if one signature is `0x12`
+and the other one is `0x23` the service should receive `0x1234`. [More information on Gnosis Safe contract signatures](contracts_signatures).
```

Comment:
Not sure about the linebreak (it is after the `0x12` which seems random). 

Also I would probably not provide the example with `0x12` and `0x23` as these are unrealistic values and it is also not always a simple concatenation, right?

Also `0x12` and `0x23` should not result in `0x1234` :D

Reply:
I removed the appending example and just left the link to the signature docs